### PR TITLE
Added Icons in .svg format

### PR DESCRIPTION
**Pull request by inakijv**

### CODE REVIEW

**Comment by inakijv** [comment]
Ping @dpobel and @SylvainGuittard

**Comment by dpobel** [comment]
great! thank you @inakijv :)

**Review by SylvainGuittard — APPROVED** [review]

File: img/Icons_svg/folder-empty.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 20.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (sunpietro) [inline img/Icons_svg/folder-empty.svg]: These comments can be stripped off the code

Comment (SylvainGuittard) [reply img/Icons_svg/folder-empty.svg]: Sure @sunpietro. We'll compress all the svg icons

**Review by sunpietro — APPROVED** [review]

**Pull request closed without merging**